test/models/spicepod_hf.yml

```diff
@@ -0,0 +1,81 @@
+version: v1beta1
+kind: Spicepod
+name: ai-test-spicepod
+
+datasets:
+  - from: s3://spiceai-demo-datasets/taxi_trips/2024/
+    name: taxi_trips
+    description: taxi trips in s3
+    params:
+      file_format: parquet
+
+  - from: github:github.com/spiceai/spiceai/issues
+    name: issues
+    description: GitHub Issues for the Spice.ai project (github.com/spiceai/spiceai)
+    metadata:
+      instructions: Always provide reference links. Use the primary key as `issue_id` to generate reference links.
+      reference_url_template: https://github.com/spiceai/spiceai/issues/<issue_id>
+    params:
+      github_token: ${secrets:GITHUB_TOKEN}
+      github_query_mode: "search"
```

Comment:
This quotes in this file needs to be updated as well

Reply:
Updated, thanks